Description: fix-libreoffice-javaldx-startup-crash
Author: Lorenzo "Palinuro" Faletra <palinuro@foxxysec.org>
Last-Update: 2017-12-29

--- firejail-0.9.52.orig/etc/libreoffice.profile
+++ firejail-0.9.52/etc/libreoffice.profile
@@ -23,12 +23,12 @@ machine-id
 netfilter
 nodvd
 nogroups
-nonewprivs
+#nonewprivs
 noroot
 notv
-protocol unix,inet,inet6
-seccomp
-shell none
+#protocol unix,inet,inet6
+seccomp.block-secondary
+#shell none
 tracelog
 
 private-dev
