Description: Import Foxxy profiles for better usability
Author: Lorenzo "Palinuro" Faletra <palinuro@foxxysec.org>
Last-Update: 2017-12-26

--- firejail-0.9.52.orig/etc/arduino.profile
+++ firejail-0.9.52/etc/arduino.profile
@@ -8,6 +8,9 @@ include /etc/firejail/globals.local
 noblacklist ${HOME}/.arduino15
 noblacklist ${HOME}/.java
 noblacklist ${HOME}/Arduino
+noblacklist ${HOME}/Downloads
+noblacklist ${HOME}/Documents
+noblacklist ${HOME}/Desktop
 
 include /etc/firejail/disable-common.inc
 include /etc/firejail/disable-devel.inc
--- firejail-0.9.52.orig/etc/bleachbit.profile
+++ firejail-0.9.52/etc/bleachbit.profile
@@ -18,7 +18,6 @@ no3d
 nodvd
 nogroups
 nonewprivs
-noroot
 nosound
 notv
 novideo
--- firejail-0.9.52.orig/etc/eom.profile
+++ firejail-0.9.52/etc/eom.profile
@@ -12,10 +12,10 @@ noblacklist ${HOME}/.config/mate/eom
 noblacklist ${HOME}/.local/share/Trash
 noblacklist ${HOME}/.steam
 
-include /etc/firejail/disable-common.inc
-include /etc/firejail/disable-devel.inc
+#include /etc/firejail/disable-common.inc
+#include /etc/firejail/disable-devel.inc
 include /etc/firejail/disable-passwdmgr.inc
-include /etc/firejail/disable-programs.inc
+#include /etc/firejail/disable-programs.inc
 
 include /etc/firejail/whitelist-var-common.inc
 
@@ -37,8 +37,8 @@ tracelog
 private-bin eom
 private-dev
 private-etc fonts
-private-lib
-private-tmp
+#private-lib
+#private-tmp
 
 memory-deny-write-execute
 noexec ${HOME}
--- firejail-0.9.52.orig/etc/firefox.profile
+++ firejail-0.9.52/etc/firefox.profile
@@ -33,6 +33,12 @@ mkdir ${HOME}/.cache/mozilla/firefox
 mkdir ${HOME}/.mozilla
 mkdir ${HOME}/.pki
 whitelist ${DOWNLOADS}
+whitelist ${HOME}/Downloads
+whitelist ${HOME}/Documents
+whitelist ${HOME}/Desktop
+whitelist ${HOME}/Pictures
+whitelist ${HOME}/Music
+whitelist ${HOME}/Video
 whitelist ${HOME}/.cache/gnome-mplayer/plugin
 whitelist ${HOME}/.cache/mozilla/firefox
 whitelist ${HOME}/.config/gnome-mplayer
--- firejail-0.9.52.orig/etc/libreoffice.profile
+++ firejail-0.9.52/etc/libreoffice.profile
@@ -8,9 +8,11 @@ include /etc/firejail/globals.local
 noblacklist ${HOME}/.java
 noblacklist /usr/local/sbin
 noblacklist ${HOME}/.config/libreoffice
+noblacklist ${HOME}/.cache
 
 include /etc/firejail/disable-common.inc
-include /etc/firejail/disable-devel.inc
+# libreoffice needs it at first run
+#include /etc/firejail/disable-devel.inc
 include /etc/firejail/disable-passwdmgr.inc
 include /etc/firejail/disable-programs.inc
 
@@ -32,5 +34,14 @@ tracelog
 private-dev
 private-tmp
 
-noexec ${HOME}
+#noexec ${HOME}
+noexec ${HOME}/Desktop
+noexec ${HOME}/Documents
+noexec ${HOME}/Downloads
+noexec ${HOME}/Music
+noexec ${HOME}/Pictures
+noexec ${HOME}/Public
+noexec ${HOME}/Templates
+noexec ${HOME}/Videos
+noexec ${HOME}/.local
 noexec /tmp
--- firejail-0.9.52.orig/etc/qbittorrent.profile
+++ firejail-0.9.52/etc/qbittorrent.profile
@@ -17,7 +17,12 @@ include /etc/firejail/disable-programs.i
 mkdir ${HOME}/.cache/qBittorrent
 mkdir ${HOME}/.config/qBittorrent
 mkdir ${HOME}/.local/share/data/qBittorrent
-whitelist  ${DOWNLOADS}
+whitelist ${DOWNLOADS}
+whitelist ${HOME}/Desktop
+whitelist ${HOME}/Documents
+whitelist ${HOME}/Pictures
+whitelist ${HOME}/Music
+whitelist ${HOME}/Video
 whitelist ${HOME}/.cache/qBittorrent
 whitelist ${HOME}/.config/qBittorrent
 whitelist ${HOME}/.config/qBittorrentrc
--- firejail-0.9.52.orig/etc/steam.profile
+++ firejail-0.9.52/etc/steam.profile
@@ -37,7 +37,7 @@ nonewprivs
 noroot
 notv
 # novideo should be commented for VR
-novideo
+#novideo
 protocol unix,inet,inet6,netlink
 seccomp
 shell none
@@ -45,7 +45,7 @@ shell none
 # tracelog
 
 # private-dev should be commented for controllers
-private-dev
+#private-dev
 # private-etc breaks some games
-#private-etc asound.conf,ca-certificates,dbus-1,drirc,fonts,group,gtk-2.0,gtk-3.0,host.conf,hostname,hosts,ld.so.cache,ld.so.preload,ld.so.conf,ld.so.conf.d,localtime,lsb-release,machine-id,mime.types,passwd,pulse,resolv.conf,ssl
+private-etc asound.conf,ca-certificates,dbus-1,drirc,fonts,group,gtk-2.0,gtk-3.0,host.conf,hostname,hosts,ld.so.cache,ld.so.preload,localtime,lsb-release,machine-id,mime.types,passwd,pulse,resolv.conf,ssl,bumblebee,alternatives,nvidia
 private-tmp
--- firejail-0.9.52.orig/etc/thunderbird.profile
+++ firejail-0.9.52/etc/thunderbird.profile
@@ -15,11 +15,12 @@ noblacklist ${HOME}/.thunderbird
 
 mkdir ${HOME}/.cache/thunderbird
 mkdir ${HOME}/.gnupg
-mkdir ${HOME}/.icedove
 mkdir ${HOME}/.thunderbird
+whitelist ${DOWNLOADS}
+whitelist ${HOME}/Desktop
+whitelist ${HOME}/Documents
 whitelist ${HOME}/.cache/thunderbird
 whitelist ${HOME}/.gnupg
-whitelist ${HOME}/.icedove
 whitelist ${HOME}/.thunderbird
 include /etc/firejail/whitelist-common.inc
 include /etc/firejail/whitelist-var-common.inc
--- firejail-0.9.52.orig/etc/uget-gtk.profile
+++ firejail-0.9.52/etc/uget-gtk.profile
@@ -13,6 +13,11 @@ include /etc/firejail/disable-programs.i
 
 mkdir ${HOME}/.config/uGet
 whitelist ${DOWNLOADS}
+whitelist ${HOME}/Desktop
+whitelist ${HOME}/Documents
+whitelist ${HOME}/Pictures
+whitelist ${HOME}/Music
+whitelist ${HOME}/Video
 whitelist ${HOME}/.config/uGet
 include /etc/firejail/whitelist-common.inc
 
--- firejail-0.9.52.orig/etc/virtualbox.profile
+++ firejail-0.9.52/etc/virtualbox.profile
@@ -21,6 +21,7 @@ mkdir ${HOME}/VirtualBox VMs
 whitelist ${HOME}/.config/VirtualBox
 whitelist ${HOME}/VirtualBox VMs
 whitelist ${DOWNLOADS}
+whitelist ${HOME}/Desktop
 include /etc/firejail/whitelist-common.inc
 include /etc/firejail/whitelist-var-common.inc
 
